pipenv/core.py

```diff
@@ -905,6 +893,21 @@ def do_install_dependencies(
         )
     if not procs.empty():
         _cleanup_procs(project, procs, failed_deps_queue, retry=False)
+    if not failed_deps_queue.empty():
+        failed_list = []
+        while not failed_deps_queue.empty():
+            failed_dep = failed_deps_queue.get()
+            failed_list.append(failed_dep)
+        click.echo(
+            "{}".format(
+                crayons.red(
+                    f"Failed to install some dependency or packages.  "
+                    f"The following have failed installation and attempted retry: {failed_list}"
+                )
```

Comment:
I'm not versed in click colors and was copying an existing pattern, but I am open to changing it here.   I think you were talking about at one point being the one that would tackle a PR to remove crayons?

Reply:
Yes. I wanted to tackle that. But instead of a big PR, I think we should simply remove this pattern slowly, and avoid it in new code. 
Click uses colorama and has easy API:
https://click.palletsprojects.com/en/7.x/utils/#ansi-colors